### PR TITLE
Add SIG Network Spotlight

### PR DESCRIPTION
**Pull request by fsmunoz**
New Spotlight article by @SD-13. Went to a preliminary round of reviews (https://github.com/SD-13/k8s-contributor-site/pull/1), including the SIG Network Chair @shaneutt.

### CODE REVIEW

File: content/en/blog/2023/sig-network-spotlight.md [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+layout: blog
+title: "Spotlight on SIG Network"
+slug: sig-network-spotlight
+date: 2023-05-09
+slug: sig-network-spotlight-2023
+author: "Sujay Dey"
+---
+
+Networking is one of the core pillars of Kubernetes, and the Special Interest
+Group for Networking(SIG Network) is responsible for developing and maintaining
```
Comment (sftim) [inline content/en/blog/2023/sig-network-spotlight.md:11]: (nit)
```suggestion
Group for Networking (SIG Network) is responsible for developing and maintaining
```

File: content/en/blog/2023/sig-network-spotlight.md [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+a bit more about SIG Network. Since we know it covers a lot of ground, could you
+please highlight its scope and current focus areas?
+
+**Shane**: For those who may be uninitiated: SIG Network is responsible for the
+components, interfaces, and APIs which expose networking capabilities to
+Kubernetes users and workloads. The [charter][net-charter] is a pretty good
+indication of our scope, but I can add some additional highlights on some of our
+current areas of focus (this is a non-exaustive list of sub-projects):
+
+**KubeProxy/KPNG**
```
Comment (sftim) [inline content/en/blog/2023/sig-network-spotlight.md:59]:
```suggesiton
**kube-proxy & KPNG**
```

File: content/en/blog/2023/sig-network-spotlight.md
Comment (sftim) [inline content/en/blog/2023/sig-network-spotlight.md:1]: Avoid writing API kinds (eg Service) in backticks.

File: content/en/blog/2023/sig-network-spotlight.md [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+various routing types (e.g. `HTTPRoute`, `GRPCRoute`, `TLSRoute`, `TCPRoute`,
+`UDPRoute`, etc.) that attach to `Gateways`. We also have an initiative within
+this project called GAMMA, geared towards using Gateway API resources in a mesh
+network context. There are some up-and-coming side projects within Gateway API
+as well, including `ingress2gateway` which is a tool for compiling existing
+`Ingress` resources to equivalent Gateway API resources and `Blixt`, a Layer4
+implementation of Gateway API using Rust/eBPF for the data plane, intended as a
+testing and reference implementation. If this sounds interesting, we would love
+to have readers join us in our Gateway API community meetings and
+`#sig-network-gateway-api` on Kubernetes Slack.
```
Comment (sftim) [inline content/en/blog/2023/sig-network-spotlight.md:116]: Maybe link to the Gateway API website, perhaps also to GAMMA?

File: content/en/blog/2023/sig-network-spotlight.md
Comment (sftim) [reply content/en/blog/2023/sig-network-spotlight.md:1]: I suggest using bold italics for topics (eg **_Gateway API_**), to differentiate them from speakers.

File: content/en/blog/2023/sig-network-spotlight.md [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+
+**Sujay**: I have to say, that was a truely inspiring journey! Now, let us talk
+a bit more about SIG Network. Since we know it covers a lot of ground, could you
+please highlight its scope and current focus areas?
+
+**Shane**: For those who may be uninitiated: SIG Network is responsible for the
+components, interfaces, and APIs which expose networking capabilities to
+Kubernetes users and workloads. The [charter][net-charter] is a pretty good
+indication of our scope, but I can add some additional highlights on some of our
+current areas of focus (this is a non-exaustive list of sub-projects):
```
Comment (sftim) [inline content/en/blog/2023/sig-network-spotlight.md:57]:
```suggestion
current areas of focus (this is a non-exhaustive list of sub-projects):
```

File: content/en/blog/2023/sig-network-spotlight.md [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+If you're specially interested in **ingress** or **mesh** networking the Gateway
+API may be a sub-project you would enjoy. In Gateway API , we're actively
+developing the successor to the illustrious `Ingress` API, which includes a
+`Gateway` resource which defines the addresses and listeners of the gateway and
+various routing types (e.g. `HTTPRoute`, `GRPCRoute`, `TLSRoute`, `TCPRoute`,
+`UDPRoute`, etc.) that attach to `Gateways`. We also have an initiative within
+this project called GAMMA, geared towards using Gateway API resources in a mesh
+network context. There are some up-and-coming side projects within Gateway API
+as well, including `ingress2gateway` which is a tool for compiling existing
+`Ingress` resources to equivalent Gateway API resources and `Blixt`, a Layer4
```
Comment (sftim) [inline content/en/blog/2023/sig-network-spotlight.md:112]: I'd write:
```markdown
Ingress objects to equivalent Gateway API resources, and Blixt, a Layer 4
```

**Review by sftim — COMMENTED** [review]
Hi. Here's some early feedback.

File: content/en/blog/2023/sig-network-spotlight.md (hunk 5 above)
Comment (fsmunoz) [reply content/en/blog/2023/sig-network-spotlight.md:112]: Done in latest commit.

File: content/en/blog/2023/sig-network-spotlight.md (hunk 3 above)
Comment (fsmunoz) [reply content/en/blog/2023/sig-network-spotlight.md:116]: Added link to Gateway API at the first reference of the it in the paragraph; GAMMA is listed under that so I didn't add it explicitly.

File: content/en/blog/2023/sig-network-spotlight.md
Comment (fsmunoz) [reply content/en/blog/2023/sig-network-spotlight.md:1]: Removed most backticks from API types, including `Pods` -> pods. 
Added italics to the project headers.

**Pull request merged**